napari_animation/_qt/animation_widget.py

```diff
@@ -103,6 +109,10 @@ def _init_save_button(self):
         self.saveButton = QPushButton('Save Animation', parent=self)
         self._layout.addWidget(self.saveButton)
 
+    def _init_scroller_interpol(self):
```

Comment:
not immediately obvious from this name what the slider does - see above re: refactor into `AnimationSliderWidget`